core/domain/entities/editor/Block.php

```diff
@@ -100,21 +79,32 @@ public function editorBlockType()
     /**
      * @return string
      */
-    public function namespacedEditorBlockType()
+    public function namespacedBlockType()
     {
-        return EditorBlock::NS . $this->editor_block_type;
+        return self::NAME_SPACE . $this->editor_block_type;
     }
 
 
     /**
      * @param string $editor_block_type
      */
-    protected function setEditorBlockType($editor_block_type)
+    protected function setBlockType($editor_block_type)
     {
         $this->editor_block_type = $editor_block_type;
```

Comment:
should this property name change (and the corresponding argument name as well)?

Reply:
yup